packages/token-search-discovery-controller/CHANGELOG.md

```diff
@@ -7,19 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.0]
+
 ### Added
 
-- Introduce the `logoUrl` property to the `TokenSearchApiService` response
+- Rename `ControllerMessenger` to `Messenger` ([#5243](https://github.com/MetaMask/core/pull/5243))
+- Bump @metamask/utils to v11.1.0 ([#5223](https://github.com/MetaMask/core/pull/5223))
```

Comment:
The "Added" category is for new features or capabilities. This bump doesn't really do that, it's more of a housekeeping item. What do you think about moving this to the "Changed" category below?